idd/Energy+.idd.in

```diff
@@ -44192,8 +44192,17 @@ ZoneHVAC:LowTemperatureRadiant:ConstantFlow,
         \key OutdoorWetBulbTemperature
         \key SurfaceFaceTemperature
         \key SurfaceInteriorTemperature
+        \key RunningMeanOutdoorAirTemperature
```

Comment:
Should this be RunningMeanOutdoorDryBulbTemperature to be more specific? Especially since OutdoorDryBulbTemperature and OutdoorWetBulbTemperature are also choices here.

Reply:
Will fix this--should show up in the next commit.